src/objects/zcl_abapgit_object_ddlx.clas.abap

```diff
@@ -279,12 +294,33 @@ CLASS ZCL_ABAPGIT_OBJECT_DDLX IMPLEMENTATION.
           TYPE ('CL_DDLX_WB_OBJECT_DATA').
 
         li_persistence = get_persistence( ).
-        li_persistence->get(
-          EXPORTING
-            p_object_key  = lv_object_key
-            p_version     = swbm_version_active
-          CHANGING
-            p_object_data = li_data_model ).
+
+        IF zcl_abapgit_factory=>get_environment( )->compare_with_inactive( ) = abap_true.
```

Comment:
will not compile until https://github.com/larshp/abapGit/pull/2809 is merged

Reply:
What's to do? Wait till #2809 is merged?

Reply:
yes

Reply:
compare_with_inactive is also a new method that is not part of #2809 ?